notebooks/exploration/utils/test_rating.py

```diff
@@ -0,0 +1,76 @@
+import pytest
```

Comment:
Some brief doc strings would be good here too. Also I'd suggest having a separate tests folder at root which contains this file